app/assets/javascripts/modules/users/components/signup-form.js

```diff
@@ -0,0 +1,36 @@
+import BaseComponent from '~/base/component';
+
+const PASSWORD_FIELD = '#user_password';
+const CONFIRMATION_PASSWORD_FIELD = '#user_password_confirmation';
+const SUBMIT_BUTTON = '#submit_btn';
+
+// UsersPasswordForm component that handles user password form
+// interactions.
+class UsersSignUpForm extends BaseComponent {
+  elements() {
+    this.$Password = this.$el.find(PASSWORD_FIELD);
+    this.$PasswordConfirmation = this.$el.find(CONFIRMATION_PASSWORD_FIELD);
+    this.$submit = this.$el.find(SUBMIT_BUTTON);
+  }
+
+  events() {
+    this.$el.on('keyup', PASSWORD_FIELD, e => this.onKeyup(e));
```

Comment:
I think the reason nothing is happening on the focus out is because is you listening to the wrong event. You should use `blur`.